src/chrome/content/rules/valgrind.org.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="valgrind.org">
```

Comment:
non-functional `njn.valgrind.org` and `svn.valgrind.org`

Reply:
Done.